src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2758,12 +2758,11 @@ private BoundExpression BindIsOperator(BinaryExpressionSyntax node, DiagnosticBa
 
             // try binding as a type, but back off to binding as an expression if that does not work.
             bool wasUnderscore = IsUnderscore(node.Right);
-            if (!TryBindAsType(node.Right, out DiagnosticBag isTypeDiagnostics, out BoundTypeExpression typeExpression) &&
+            if (!tryBindAsType(node.Right, out DiagnosticBag isTypeDiagnostics, out BoundTypeExpression typeExpression) &&
```

Comment:
>tryBindAsType [](start = 17, length = 13)

just to confirm: is the `TryBindAsType` method (which we previously used) still used elsewhere?

Reply:
That method has been deleted and changed to be a local function here.

---
In reply to: [362988396](https://github.com/dotnet/roslyn/pull/40503#discussion_r362988396) [](ancestors = 362988396)